### PR TITLE
Update deegree webservices references

### PR DESCRIPTION
**Pull request by julianzz98**
This pull request updates the deegree CITE reference implementation to the version of 3.5.0 and updates absolute version links to a relative "current (version)" one. 

Furthermore a typo was resolved, as seen in http://opengeospatial.github.io/e-learning/wfs/text/operations.html#propetyname-parameter

### CODE REVIEW

**Review by ghobona — APPROVED** [review]

**Review by bradh — COMMENTED** [review]
This line is not formatted correctly.

**Comment by julianzz98** [comment]
> This line is not formatted correctly.

Can you specify the file and line again? Not seeing where your annotation is located right now.

File: source/wfs/text/operations.rst [hunk 1]
```diff
@@ -358,7 +358,7 @@ The response returned by the request above is shown below. The response is prese
   </wfs:member>
   </wfs:FeatureCollection>
 
-PropetyName parameter
+PropertyName parameter
 ^^^^^^^^^^^^^^^^^^^^^
```
Comment (bradh) [inline source/wfs/text/operations.rst:362]: This line.

Comment (julianzz98) [reply source/wfs/text/operations.rst:362]: Thank you! Will fix it and create a new PR, my bad!

**Pull request merged**